src/test/app/Freeze_test.cpp

```diff
@@ -1778,6 +1781,151 @@ class Freeze_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testNFTOffersWhenFreeze(FeatureBitset features)
```

Comment:
these tests only exercise when someone accepts a sell offer from the owner. have we considered adding tests where the NFT owner accepts a buy offer? (where the potential buyer is deep frozen)

Reply:
i thought of another scenario:
1. alice mints NFT
2. alice creates a sell offer using USD
3. bob creates a buy offer using USD
4. carol (broker) accepts the two offers and earns some USD (if buy offer amount is higher than sell offer, broker has the oppportunity to earn `buy offer amt - sell offer amt`) You can find unit tests by searching for `brokerOffers`.

we would need to make sure that at step 4, carol cannot broker the offers if she is deep frozen